helpers/transformers/web.js

```diff
@@ -38,8 +36,9 @@ function convertFonts(fonts) {
  * Transform the tokens into a web specific object that can be used in emotion
  *
  * @param  {string} BRAND - The brand string to find the right brand folder
+ * @param  {string} dest - The destination path where the file should be written
  */
-function build(BRAND) {
+function build(BRAND, dest = '.') {
```

Comment:
Passed a default value of current directory here to preserve the existing behaviour if no second argument is passed - not sure if this is the best way to approach it.

Reply:
dest isn't necessary since if you know the brand, you also know the directory.